app/views/admin/navigation_links/_add_navigation_link_modal.html.erb

```diff
@@ -1,18 +1,16 @@
-<span data-controller="modal" data-modal-root-selector-value="#add-navigation-link-modal-root" data-modal-content-selector-value="#add-navigation-link-modal" data-modal-title-value="Add Navigation Link" data-modal-size-value="l">
-  <button class="btn btn-primary" type="button" data-action="modal#toggleModal" id="nav-link-button">
-    Add Navigation Link
-  </button>
+<div data-controller="modal" data-modal-root-selector-value="#add-navigation-link-modal-root" data-modal-content-selector-value="#add-navigation-link-modal" data-modal-title-value="Add Navigation Link" data-modal-size-value="medium">
+  <div class="border-solid border-b-1 border-base-10 border-0 pb-6">
+    <button class="c-btn c-btn--secondary" type="button" data-action="modal#toggleModal" id="nav-link-button">
+      Add navigation link
```

Comment:
The change from `Add Navigation Link` to `Add navigation link` is causing one of the E2E tests to fail - if you dip into `seededFlows/adminFlows/navigationLinks/navigationLinks.spec.js` and change the name it should all go green again 🚀